index.html

```diff
@@ -22,21 +22,109 @@ <h1>Lucky Duck Restaurang</h1>
   <div class="allergens-icon"></div>
 
  </div><!--End of Navbar-->
- <!------------------------------------------------------------------>
+  <!------------------------------------------------------------------>
+  
  <div class="menu-component">
-  <div class="meal-1">
-
+  <!--add more meal from here, just copy this div -->
+  <!------------------meal1----------------->
+  <div class="meal">
+   <div class="menu-text">
```

Comment:
There's only supposed to be one language displayed at once. The user will have the option to switch languages in the navigation bar, as show on Trello. @dshertz has made some commits to his branch, which shows the language switcher in the navigation bar. 

I think it would be better to crop the image, to not include the darker lines in the image, as it makes the text harder to read. Below is an example of how that could look. The left is the original, while the one on the right is the example 
![image](https://user-images.githubusercontent.com/36706634/195940435-7e4c8577-91a6-4e5a-abae-ed282971f1f1.png)